apps/shared/test-helpers/mine.js

```diff
@@ -1,43 +1,49 @@
-const Web3 = require("web3");
-const PrivateKeyProvider = require("truffle-privatekey-provider");
-const provider = new PrivateKeyProvider(
-    "A8A54B2D8197BC0B19BB8A084031BE71835580A01E70A45A13BABD16C9BC1563",
-    "http://localhost:8545"
-  );
-const web3 = new Web3(provider);
+/// uncomment the Below block to use on aragon devchain without truffle
 
-const mineBlock = require("./mineBlock")(web3)
-const getBlock = require("./blockNumber")(web3)
+//const Web3 = require("web3");
+//const PrivateKeyProvider = require("truffle-privatekey-provider");
+//const provider = new PrivateKeyProvider(
+//    "A8A54B2D8197BC0B19BB8A084031BE71835580A01E70A45A13BABD16C9BC1563",
+//    "http://localhost:8545"
+//  );
+//const web3 = new Web3(provider);
 
-const mineBlocks = async (blocks) => {
-    for (var i = 0; i < blocks; i++) {
-        try {
-            await mineBlock()
-        }
-        catch(e) {
-            console.error('error: ',e)
+const mineBlock = require("./mineBlock")
+const getBlock = require("./blockNumber")
+
+module.exports = async (cb) => {
+    const mineBlockFn = mineBlock(web3)
```

Comment:
The indentation in this file differs from our others. Why is that?

More importantly, where does the `web3` variable come from, when the above code is commented out? If it's a global variable that's available thanks to Truffle, then instead of requiring someone to switch the commented code above, can we check if `web3` is defined and use the new style if so, and the old style if not?

Reply:
The indendation is different from our app files, but it appears that the helper functions in the library are all node scripts with 4-spaced indentations